File: Cap 04/Topico 4.5/Topico 4.5.3/Decisao por Selecao/decsel.java
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.5.2
 * 
 * Programa ..: OPERACAO COM DECISAO SELETIVA
 */

import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    int MES;
    Scanner entrada = new Scanner(System.in);
    System.out.println("MES POR EXTENSO\n");
    System.out.print("Entre o numero do mes: ");
    MES = entrada.nextInt();
      switch (MES) {
      case 1:
        System.out.println("Janeiro");
        break;
      case 2:
        System.out.println("Fevereiro");
        break;
      case 3:
        System.out.println("Marco");
        break;
      case 4:
        System.out.println("Abril");
        break;
      case 5:
        System.out.println("Maio");
        break;
      case 6:
        System.out.println("Junho");
        break;
      case 7:
        System.out.println("Julho");
        break;
      case 8:
        System.out.println("Agosto");
        break;
      case 9:
        System.out.println("Setembro");
        break;
      case 10:
        System.out.println("Outubro");
        break;
      case 11:
        System.out.println("Novembro");
        break;
      case 12:
        System.out.println("Dezembro");
        break;
      default:
        System.out.println("Valor invalido");
        break;
    }
    entrada.close();
  }
}

File: Cap 04/Topico 4.5/Topico 4.5.3/Decisao por Selecao/decsel.cpp
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 4 - Topico 4.5.Programacao
 * 
 * Programa ..: OPERACAO COM DECISAO SELETIVA
 * 
 */

#include <iostream>

using namespace std;

int MES;
int main(void) {
  cout << "MES POR EXTENSO" << endl << endl;
  cout << "Entre o numero do mes: ";
  cin >> MES;
  switch (MES) {
    case 1:
      cout << "Janeiro" << endl;
      break;
    case 2:
      cout << "Fevereiro" << endl;
      break;
    case Programacao:
      cout << "Marco" << endl;
      break;
    case 4:
      cout << "Abril" << endl;
      break;
    case 5:
      cout << "Maio" << endl;
      break;
    case 6:
      cout << "Junho" << endl;
      break;
    case 7:
      cout << "Julho" << endl;
      break;
    case 8:
      cout << "Agosto" << endl;
      break;
    case 9:
      cout << "Setembro" << endl;
      break;
    case 10:
      cout << "Outubro" << endl;
      break;
    case 11:
      cout << "Novembro" << endl;
      break;
    case 12:
      cout << "Dezembro" << endl;
      break;
    default:
      cout << "Valor invalido" << endl;
      break;
  }
  return 0;
}
